docs/en/observability/apm/view-and-analyze/drill-down/infrastructure.asciidoc

```diff
@@ -4,11 +4,28 @@
 beta::[]
 
 The *Infrastructure* tab provides information about the containers, pods, and hosts
-that the selected service is linked to.
+that the selected service is linked to:
+
+// Does it make sense to add this here?
+* *Pods*: Uses the `kubernetes.pod.name` from the <<apm-data-model-metrics,APM metrics data streams>>.
```

Comment:
I think it's good information to know which fields are queried to get the infrastructure data in APM